common/src/main/java/org/opensearch/ml/common/connector/MLPostProcessFunction.java

```diff
@@ -5,61 +5,64 @@
 
 package org.opensearch.ml.common.connector;
 
+import org.opensearch.ml.common.output.model.MLResultDataType;
+import org.opensearch.ml.common.output.model.ModelTensor;
+
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.function.Function;
 
 public class MLPostProcessFunction {
 
-    private static Map<String, String> POST_PROCESS_FUNCTIONS;
     public static final String COHERE_EMBEDDING = "connector.post_process.cohere.embedding";
     public static final String OPENAI_EMBEDDING = "connector.post_process.openai.embedding";
 
+    public static final String NEURAL_SEARCH_EMBEDDING = "connector.post_process.default";
```

Comment:
IS this default post process function will only be for embedding? If yes, then may be we can say: `connector.post_process.default.embedding`?

and the static variable name can be: `DEFAULT_EMBEDDING` (to match other two?)